### ISSUE TITLE
Spacing between words

### ISSUE THREAD

**Issue by suhasdarsi** [tokotype/PlusJakartaSans#25]
I have no understand about principles of font building so, feel free ignore this if it doesn't make any sense. I have noticed that while trying to use the font for real world I found the space between words a little inadequate 
![Frame 1000001958](https://user-images.githubusercontent.com/10138686/159060531-e70e5467-2fdc-466f-9775-cbde162d4a6b.png)

In this image the fonts are as follows
- Plus Jakarta
- Circular Std
- DM Sans

I think the others do a better job at the spacing. It seems like a 1.25x increase in the distance could do wonders in terms of readability.

**Comment by amoghsrivastava** [comment]
I agree! The readability in smaller font settings is a little tricky. The spacing in between letters is slightly more and the spacing between words is a slightly. That kinda leads to difficulty in readability. It doesn't feel as compact and cohesive.

**Comment by thedesignchild** [comment]
Looking forward to this being fixed. One of the only reasons stopping me from using Plus Jakarta 🥲
